### PR TITLE
Release PR

### PR DESCRIPTION
**Pull request by a-ramit**
This is a release PR for Alpha release.

### CODE REVIEW

**Comment by a-ramit** [comment]
Hi @tcchiba - please review the Release PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changelog Short Message Service
+
+# v0.1.0 - Initial version
```
Comment (hdamker) [inline CHANGELOG.md:3]: It's your very first release. Release will be number sequentially (r0.1, r0.2, ...)

```suggestion
# r0.1
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+**Initial version of SMS.YAML and sms-delivery-notification-subscription.yaml.**
```
Comment (hdamker) [inline CHANGELOG.md:5]: An "Initial" version within the [API Versioning](https://wiki.camaraproject.org/x/a4BaAQ) nomenclature will be later public-release versions 0.1.0 (after alpha and rc phase). Here you have the first alpha versions of that.
```suggestion
** This release contains the first alpha versions of SMS.YAML 0.1.0 and sms-delivery-notification-subscription.yaml 0.1.0**
```

**Review by hdamker — COMMENTED** [review]
Looks good, but please have a look on my comments.

And remember to create the release only as "pre-release".

**Review by tcchiba — APPROVED** [review]

**Review by tcchiba — APPROVED** [review]

**Review by tcchiba — APPROVED** [review]

**Review by tcchiba — APPROVED** [review]

**Pull request merged**